src/Syntax.php

```diff
@@ -20,6 +21,12 @@ public function __construct($openingTag = null, $closingTag = null, $closingTagM
         $this->closingTagMarker = $closingTagMarker ?: '/';
         $this->parameterValueSeparator = $parameterValueSeparator ?: '=';
         $this->parameterValueDelimiter = $parameterValueDelimiter ?: '"';
+
+        $this->openingTag = trim($this->openingTag);
+        $this->closingTag = trim($this->closingTag);
+        $this->closingTagMarker = trim($this->closingTagMarker);
+        $this->parameterValueSeparator = trim($this->parameterValueSeparator);
+        $this->parameterValueDelimiter = trim($this->parameterValueDelimiter);
```

Comment:
This block could be merged with original block above to `trim()` and handle default values in one line.